packages/components/pages/f-takeawaypay-activation/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-takeawaypay-activation",
   "description": "Fozzie TakeawayPay Activation - Handles Takeaway Pay activation for users",
-  "version": "2.9.0",
+  "version": "2.10.0",
```

Comment:
I think this should be a major version bump, as effectively the change in shape of the logger object this component consumes is a backwards compatibility break.